app/models/AppDatabase.scala

```diff
@@ -1,26 +1,46 @@
 package models
 
+import javax.inject.{Inject, Singleton}
+
 import com.outworkers.phantom.builder.query.CreateQuery
-import com.outworkers.phantom.connectors.{ ContactPoint, CassandraConnection }
 import com.outworkers.phantom.dsl._
-import scala.concurrent.duration._
 
-object Defaults {
-  val connector: CassandraConnection = ContactPoint.local.keySpace("outworkers")
-}
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
 
-class AppDatabase(val keyspace: CassandraConnection) extends Database[AppDatabase](keyspace) {
+@Singleton
+class AppDatabase @Inject()(env: play.api.Environment, override val connector: CassandraConnection) extends Database[AppDatabase](connector) {
 
-  object users extends Users with Connector {
-    override def autocreate(space: KeySpace): CreateQuery.Default[Users, User] = {
+  object beers extends Beers with Connector {
+    override def autocreate(space: KeySpace): CreateQuery.Default[Beers, Beer] = {
       create.ifNotExists()(space)
         .option(default_time_to_live eqs 10)
         .and(gc_grace_seconds eqs 10.seconds)
         .and(read_repair_chance eqs 0.2)
     }
-  }
-  object beers extends Beers with Connector
-}
 
+    def save(beer: Beer): Future[ResultSet] = {
```

Comment:
This method is macro generated on the table, `beers.storeRecord(beer)` should be more than enough here.

Reply:
I can't find any documentation on storeRecord, and when I try using it, it tells me it can't find an implicit SingleGeneric.Aux.  I have looked at https://github.com/outworkers/phantom/wiki/Using-the-Database-class-and-understanding-connectors and that uses an insert...

Reply:
@wsargent I see, that is documented [here](https://github.com/outworkers/phantom/blob/develop/docs/basics/tables.md#automatically-derived-store-method), I will move this to a separate page and make it more obvious.

Reply:
I don't see `storeRecord` on https://github.com/outworkers/phantom/blob/develop/docs/basics/tables.md#automatically-derived-store-method either, only `store`

Reply:
```
[play-phantom] $ test
[info] Compiling 1 Scala source to /Users/wsargent/work/phantom-activator-template/target/scala-2.12/classes...
[error] /Users/wsargent/work/phantom-activator-template/app/models/AppDatabase.scala:34: could not find implicit value for parameter sg: com.outworkers.phantom.macros.SingleGeneric.Aux[models.Beer,Repr,HL,Out]
[error]         records <- beers.storeRecords(beerList)
[error]                                      ^
[error] one error found
[error] (compile:compileIncremental) Compilation failed
[error] Total time: 4 s, completed Sep 29, 2017 7:25:16 AM
```